meetings/agenda.md

```diff
@@ -10,3 +10,23 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 ### Abandon `mezzio/mezzio-aurarouter`
 
 As per [the RFC](https://github.com/mezzio/mezzio-aurarouter/issues/12), should we abandon this package and archive it?
+
+### Maintenance Chores
+
+#### Old Release Branch Clean Up
+
+Packages like `laminas-validator` have a _lot_ of branches. Many of these branches will never see any more commits, and it is trivial to re-create those branches if we ever need to add a patch to an old release.
+
+Is it acceptable to delete release branches older than say 1 year?
+
+#### Old Milestone Clean Up
+
+Similar to the above, deleting _(not closing 😂)_ old patch milestones. Acceptable?
```

Comment:
I have done this time and time again, so yes! 😄 
(Compare [laminas-validator](https://github.com/laminas/laminas-validator/milestones) with [laminas-mvc-plugin-identity](https://github.com/laminas/laminas-mvc-plugin-identity/milestones))